### PR TITLE
Update localization

### PR DESCRIPTION
**Pull request by Adaptivity**
Changed localization to be like in Blood Magic.

### CODE REVIEW

File: src/main/resources/assets/forbidden/lang/ru_RU.lang [hunk 1]
```diff
@@ -267,6 +272,18 @@ forbidden.research_text.TAINTBASICS=Глубокий пурпур
...
 forbidden.research_page.TAINTBASICS.2=Заражённые земли можно найти в природе, но также, они могут образовываться при неосторожности тауматурга, если заражённая слизь простоит слишком долго без внимания. Заражённые земли узнаваемы из-за их выдающихся фиолетовых волокон и ярких фиолетовых растений. Оставшись без внимания, заражённая земля будет постепенно расширять свои владения, заражая всё больше и больше окрестных земель. Земля становится густой, а заражённая почва, деревья и другие природные материалы медленно покрываются волокнами и преобразуются в структурно нездоровую кору, которая может рухнуть или раствориться в заражённую слизь в любой момент.
 #
+forbidden.research_name.SCHOOLS=Школы магии
+forbidden.research_text.SCHOOLS=Существует больше одного путя к энергии...
```
Comment (fewizz) [inline src/main/resources/assets/forbidden/lang/ru_RU.lang]: может "пути"?

**Comment by SpitefulFox** [comment]
Why did you change the Collar of Pain's tooltip?  It's intentionally not meant to sound like the Blood Magic items.

**Comment by Adaptivity** [comment]
I'm literally changed nothing. I've just moved spaces to code by adding + " ". So, spaces no longer required in .lang file.

**Comment by SpitefulFox** [comment]
Oops, sorry.  Misread the changes.  I just woke up. xP

**Comment by Adaptivity** [comment]
It's ok :)

**Pull request merged**